src/backend/env/mod.rs

```diff
@@ -1061,7 +1046,7 @@ impl State {
     pub fn minting_ratio(&self) -> u64 {
         let circulating_supply: Token = self.balances.values().sum();
         let factor = (circulating_supply as f64 / CONFIG.maximum_supply as f64 * 10.0) as u64;
-        1 << factor
+        (1 << factor) * CONFIG.difficulty_amplification
```

Comment:
Wdyt about rendering this on the token page as the minting ratio?

For example:
- Instead of `24 : 1`
- Show `(4 * 6) : 1`

Reply:
I would not invest time into this as I consider this only a workaround and already started a new discussion for a proper solution: https://6qfxa-ryaaa-aaaai-qbhsq-cai.icp0.io/#/post/849195 Btw it would be great if you would share your thoughts as well.

Reply:
Sounds good.